.github/workflows/asset-build-and-publish.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       matrix:
         # NOTE: Hard Coded Node Version array, should match array in test-asset.yml
-        node-version: [18.19.1, 20.11.1, 22.2.0]
+        node-version: [18, 20, 22.4.1]
```

Comment:
22.5.1 is out and works for me locally. I think we should unpin this as well.

Reply:
Yeah, if it works locally, it makes sense to try and unpin ...

Reply:
I'll update the node-base PR as well. I've tested the build step locally and seems to work